multiqc/utils/search_patterns.yaml

```diff
@@ -203,6 +203,8 @@ star:
     fn: '*Log.final.out'
 star_genecounts:
     fn: '*ReadsPerGene.out.tab'
+sortmerna:
+    contents: 'Command: sortmerna'
```

Comment:
This doesn't work with the example data in the [MultiQC_TestData](https://github.com/ewels/MultiQC_TestData/blob/master/data/modules/SortMeRNA/sortmerna.log) repo, as the command can have the full path to `sortmerna`. The above has:
```
Command: /home/lina/sortmerna/sortmerna
```

I guess just `contents: 'sortmerna'` should be sufficiently specific?